provision/rosa-cross-dc/Taskfile.yaml

```diff
@@ -30,35 +30,162 @@ includes:
 
-  create-peering-connection:
+  create-single-peering-connection:
+    internal: true
     desc: "Creates Aurora Peering Connection"
+    silent: false
+    requires:
+      vars:
+        - ROSA_CLUSTER_NAME
+        - AURORA_CLUSTER
+    vars:
+      AURORA_REGION: "$(cat .task/rosa-region-{{.ROSA_CLUSTER_NAME}})"
+    cmd: CLUSTER_NAME="{{.ROSA_CLUSTER_NAME}}" AURORA_REGION="{{ .AURORA_REGION }}" KUBECONFIG="{{.ISPN_DIR}}/.task/kubecfg/{{.ROSA_CLUSTER_NAME}}" SKIP_ROSA_LOGIN=true {{.RDS_DIR}}/aurora_create_peering_connection.sh
+    preconditions:
+      - test -f {{.ISPN_DIR}}/.task/kubecfg/{{.ROSA_CLUSTER_NAME}}
+
+  create-global-peering-connection:
+    internal: true
+    desc: "Creates Global Aurora Peering Connections"
+    silent: false
+    requires:
+      vars:
+        - ROSA_CLUSTER_NAME
+        - AURORA_CLUSTER
+    cmd: CLUSTER_NAME="{{.ROSA_CLUSTER_NAME}}" AURORA_GLOBAL_CLUSTER="{{.AURORA_CLUSTER}}" KUBECONFIG="{{.ISPN_DIR}}/.task/kubecfg/{{.ROSA_CLUSTER_NAME}}" SKIP_ROSA_LOGIN=true {{.RDS_DIR}}/aurora_create_global_peering_connections.sh
+    preconditions:
+      - test -f {{.ISPN_DIR}}/.task/kubecfg/{{.ROSA_CLUSTER_NAME}}
+
+  delete-single-peering-connection:
+    internal: true
+    desc: "Deletes Aurora Peering Connection"
+    silent: false
+    requires:
+      vars:
+        - ROSA_CLUSTER_NAME
+        - AURORA_CLUSTER
+    vars:
+      AURORA_REGION: "$(cat .task/rosa-region-{{.ROSA_CLUSTER_NAME}})"
+    cmds:
+      - task: fetch-rosa-region
+        vars:
+          ROSA_CLUSTER_NAME: "{{.ROSA_CLUSTER_NAME}}"
+      - cmd: CLUSTER_NAME="{{.ROSA_CLUSTER_NAME}}" AURORA_CLUSTER="{{ .AURORA_CLUSTER }}" AURORA_REGION="{{ .AURORA_REGION }}" KUBECONFIG="{{.ISPN_DIR}}/.task/kubecfg/{{.ROSA_CLUSTER_NAME}}" SKIP_ROSA_LOGIN=true {{.RDS_DIR}}/aurora_delete_peering_connection.sh
+    preconditions:
+      - test -f {{.ISPN_DIR}}/.task/kubecfg/{{.ROSA_CLUSTER_NAME}}
+
+  delete-global-peering-connection:
     internal: true
+    desc: "Deletes Global Aurora Peering Connections"
     silent: false
     requires:
       vars:
         - ROSA_CLUSTER_NAME
         - AURORA_CLUSTER
-        - AURORA_REGION
-    cmd: CLUSTER_NAME="{{.ROSA_CLUSTER_NAME}}" KUBECONFIG="{{.ISPN_DIR}}/.task/kubecfg/{{.ROSA_CLUSTER_NAME}}" SKIP_ROSA_LOGIN=true {{.RDS_DIR}}/aurora_create_peering_connection.sh
+    cmd: CLUSTER_NAME="{{.ROSA_CLUSTER_NAME}}" AURORA_GLOBAL_CLUSTER="{{.AURORA_CLUSTER}}" KUBECONFIG="{{.ISPN_DIR}}/.task/kubecfg/{{.ROSA_CLUSTER_NAME}}" SKIP_ROSA_LOGIN=true {{.RDS_DIR}}/aurora_delete_global_peering_connections.sh
     preconditions:
       - test -f {{.ISPN_DIR}}/.task/kubecfg/{{.ROSA_CLUSTER_NAME}}
 
-  fetch-aurora-endpoint:
+  fetch-global-aurora-region:
+    internal: true
+    requires:
+      vars:
+        - ROSA_CLUSTER_NAME_1
+        - ROSA_CLUSTER_NAME_2
+    cmd: ../aws/rds/aurora_endpoint.sh > .task/aurora-endpoint-{{.AURORA_CLUSTER}}
```

Comment:
@mhajas looks like we need to set this value to `KC_DATABASE_URL=$AURORA_GLOBAL_CLUSTER.aurora-global.keycloak-benchmark.com` 
as per the docs 

```
In order to ensure that the aforementioned Route53 entry reflects the writer endpoint of the Primary cluster after failover, we deploy an AWS Lambda function to each of the $AURORA_GLOBAL_REGIONS. This function is triggered on completion of a global-failover event, in the region of the new Primary cluster, and updates the CNAME entry to point to the latest writer endpoint.
```
and its also reflecting in line 70 of lambda.py

Reply:
fixed the issue in the latest commit.